api/db-cdr/csv/domain_to_domain_id.csv

```diff
@@ -0,0 +1,15 @@
+"domain","domain_id"
+0,"Condition"
```

Comment:
I don't understand how these Condition/Observation type domains are used in workbench or whatever. There are no concepts in concept table that have Condition/Observation domain id. But we do have counts like for concept id :  141693  that came from Condition table and the Observation table. Do : select * from cdr20180913.achilles_results where stratum_1 = '141693' 
on test db . 

If you want to know what bq tables to search when you pick a concept id -- you should look at that the achilles_results analysis_id 3000 for that concept_id and it will tell you what tables you will find rows in.

Reply:
After discussion offline: we're going to ignore the domain IDs with '/' in them for now. So I'm going to get rid of this file, and the logic in concept search that uses mappings like these.